test/end-to-end/helpers/mvCaseHelper.js

```diff
@@ -109,93 +153,108 @@ async function multiMediaAudioPauseAndRewindTest(I, caseId, mediaType) {
 }
 
 async function highlightTextTest(I, caseId, mediaType) {
-  await openCaseDocumentsInMediaViewer(I, caseId, mediaType);
-  await I.highlightPdfText();
+  if (process.env.TEST_URL.split('-')[3] !== 'pr' || await getEnvironment() !== 'local') {
+    await openCaseDocumentsInMediaViewer(I, caseId, mediaType);
+    await I.highlightPdfText();
+  }
 }
 
 async function addCommentTest(I, caseId, mediaType) {
-  await openCaseDocumentsInMediaViewer(I, caseId, mediaType);
-  await I.addComments();
+  if (process.env.TEST_URL.split('-')[3] !== 'pr' || await getEnvironment() !== 'local') {
```

Comment:
I think this can be a controlled by a new var and/or this can be function instead of repeating this block.